### PR TITLE
P0 - Image Component

### PR DESCRIPTION
**Pull request by bharati-21**
- Styled image component with the following variations:
  - simple responsive image
  - responsive image with rounded corners
  - round image

### CODE REVIEW

**Comment by poojavishnoi** [comment]
UI looks great! I don't think I have suggestions, it's very nice:)

**Review by shashikantmaurya0007 — APPROVED** [review]
code looks clean .very wise use of variables for repeated value.UI of the page looks beautiful:)

**Comment by parthpandyappp** [comment]
Looks cool, fine to go :+1:

File: styles.css [hunk 1]
```diff
@@ -200,12 +201,18 @@ hr.component__separator {
 .code-display iframe{
     width: 100%;
     border: none;
+    border:0;
```
Comment (TejasShekar) [inline styles.css]: line 203 has border:none & you've given border:0 right after it. Please look into it unless it has been done intentionally.

**Review by TejasShekar — COMMENTED** [review]
Like the way you've used variables. The code looks.

**Comment by bharati-21** [comment]
> UI looks great! I don't think I have suggestions, it's very nice:)

Thank you for your review and time!

**Comment by bharati-21** [comment]
> code looks clean .very wise use of variables for repeated value.UI of the page looks beautiful:)

Thank you for your review and time!

**Comment by bharati-21** [comment]
> Looks cool, fine to go 👍

Thank you for your review and time!

File: styles.css (hunk 1 above)
Comment (bharati-21) [reply styles.css]: I did this by mistake. I will remove `border: 0;`. Thank you for pointing it out.

**Pull request merged**